modules/ROOT/pages/_partials/sync/server-deploying-your-app-built-using-the-data-sync-framework-to-a-development-environment.adoc

```diff
@@ -0,0 +1,275 @@
+[id="server-deploying-your-app-built-using-the-data-sync-framework-to-a-development-environment-{context}"]
+= Deploying your app built using the data sync framework to an environment
+
+This section describes how to deploy an app using the Data Sync framework to an environment.
```

Comment:
Question: why the wording `Deploying your app built using the data sync framework *to an environment*?

* What is 'an environment?'
* It's clearly an OpenShift environment so why is it not stated?
* How is a user supposed to know what 'an environment' is? / Where is it established in our docs that 'an environment' means OpenShift?

As a developer that consumes a lot of documentation, this wording is unintuitive and sets off alarm bells about the product in my head.

Reply:
@darahayes, I would tend to agree with you. I put this piece of work to colleagues in my organization (CCS) and the general consensus was that it could be better in a more descriptive sense. "OpenShift" was proposed. @wei-lee , what are your thoughts on this?

Reply:
Deploying a Data Sync server on OpenShift

Reply:
https://github.com/aerogear/mobile-docs/pull/694/commits/7077ea55dbeb28bc7ac71a78d4282ee2c8ddd782